packages/react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -122,6 +123,9 @@ export interface DropdownProps
   /** A dropdown can be formatted to appear inline next to other elements. */
   inline?: boolean
 
+  /** It can be on a dark background */
+  inverted?: boolean
```

Comment:
Consistent description with other inverted props. Also, please see how the other props descrption are defined in the form of `A dropdown can ....` 

Try to follow that format